app/runner/runner.go

```diff
@@ -135,10 +140,17 @@ func (r *OrbitRunner) run(task *orbitTask) error {
 	for _, cmd := range task.Run {
 
 		var e *exec.Cmd
-		if runtime.GOOS == "windows" {
-			e = exec.Command(os.Getenv(defaultWindowsShellEnvVariable), "/c", cmd)
+		if task.Shell != "" {
+			shellAndParams := strings.Fields(task.Shell)
+			shell := shellAndParams[0]
+			parameters := append(shellAndParams[1:], cmd)
```

Comment:
I think we should check the numbers of field (if == 1, this won't work)

Reply:
Not sure about that `ariane` test works correctly, right?

Reply:
Indeed... missed it!